src/app/zap-templates/zcl/data-model/chip/concentration-measurement-cluster.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP" />
+  <cluster>
+    <name>Concentration Measurement</name>
+    <domain>Measurement &amp; Sensing</domain>
+    <description>Attributes and commands for configuring the measurement of concentration, and
+      reporting concentration measurements.</description>
+    <code>0x040C</code>
```

Comment:
This is the Carbon Monoxide Concentration Measurement cluster (not clearly labeled so in the spec, sadly), not the generic Concentration Measurement.